forestry/operations.py

```
@@ -49,14 +45,13 @@ def print_volume(stand: ForestStand) -> ForestStand:
     'basal_area_thinning': basal_area_thinning,
     'stem_count_thinning': stem_count_thinning,
     'continuous_growth_thinning': continuous_growth_thinning,
-    'reporting': reporting,
-    'print_volume': print_volume
+    'report_volume': report_volume
 }
 
 
 try:
     from forestry.grow_motti import grow_motti
-except ImportError:
+except Exception:
     # just don't register it when pymotti isn't found.
     # we don't want to make pymotti a required dependency until it's public.
     pass
```

Why is the exception type changed here? If the import statement raises anything other than an `ImportError`, then it's probably a bug in pymotti, and it's more useful to show a traceback than silently ignore it. (or, as in the teams thread, it could be caused by having an empty `pymotti` directory in the root, but that shouldn't happen anymore since it's now in `external/pymotti`)

---

Thanks, good catch. Yeah this was a knee-jerk reaction to errors caused by the said namespace ambiguity. Accidentally included in commit. Reverted, and a non-issue now with later changes.